Developers/Newcomers_Guide.rst

```diff
@@ -184,3 +184,8 @@ they are really rewarding!
 
     If you need help picking up an issue, you can always ask us and we'll help
     you!
+
```

Comment:
imo this should be set not in the end but somewhere around "Sending your changes" at line 108.